src/phasorpy/phasor.py

```diff
@@ -1113,6 +1116,28 @@ def phasor_calibrate(
             f'reference_real.shape={ref_re.shape} '
             f'!= reference_imag.shape{ref_im.shape}'
         )
+
+    skip_axis, axis = _parse_skip_axis(skip_axis, re.ndim)
+    if re.ndim > 0:
+        harmonic, has_harmonic_axis = parse_harmonic(harmonic, re.shape[0] * 2)
+    else:
+        has_harmonic_axis = False
+        harmonic = 1 if harmonic is None else harmonic
+
+    frequency = numpy.asarray(frequency)
+    frequency = frequency * harmonic
+
+    if has_harmonic_axis:
```

Comment:
0 should be prepended to `skip_axis` only if it has harmonic axis, right? Because if not we would then skip the first axis in the cases were we dont have harmonics.

Reply:
Yes, that comment was supposed to be under `if has_harmonic_axis:`